src/modules/edc-demo/components/error-state/error-state.component.ts

```diff
@@ -0,0 +1,16 @@
+import {Component, HostBinding, Input} from '@angular/core';
+
+@Component({
+  selector: 'edc-demo-error-state',
+  templateUrl: './error-state.component.html',
+})
+export class ErrorStateComponent {
+  @HostBinding('class.flex')
+  @HostBinding('class.flex-col')
+  @HostBinding('class.justify-center')
+  @HostBinding('class.h-[700px]')
+  cls = true;
+
+  @Input()
+  data = '';
```

Comment:
Same as for empty message, data is too generic and should be avoided where possible. It should be called "errorMessage".

It should also have a sensible default, e.g. 'Failed fetching data.'